cookbooks/fb_storage/recipes/default.rb

```diff
@@ -108,7 +110,7 @@
                  'It must be a number with an optional suffix of %%kmgt'
           %w{start end}.each do |disp|
             unless partition["partition_#{disp}"].match(
-              /^\d+([KkMmGgTt\%](iB)?)?$/,
+              /^\d+(\.\d+)?([KkMmGgTt\%](iB)?)?$/,
```

Comment:
Can you elaborate on this?

Reply:
I did, in the description. It allows for decimals. `6.5G` :)

Reply:
Oh LOL it says "partial fractions" I meant "partial sizes" or "fraction sizes" heh.